docker/scripts/scripts/httpsender/Alert_on_Unexpected_Content_Types.js

```diff
@@ -44,7 +31,7 @@ function responseReceived(msg, initiator, helper) {
 			if (ctype.indexOf(";") > 0) {
 				ctype = ctype.substring(0, ctype.indexOf(";"))
 			}
-			if (expectedTypes.indexOf(ctype) < 0) {
+			if (!msg.getResponseHeader().hasContentType(expectedTypeGroups) || expectedTypes.indexOf(ctype) < 0) {
```

Comment:
`&&`

Reply:
Why and, we only expect one to be hit.

Reply:
If either not one of the type groups or not one of the specific types, raise an alert.

Reply:
We need both false not just one or the other.

Reply:
UGH, yup....logic fail 😞